spec/cli_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require File.expand_path('spec_helper', __dir__)
+require 'undercover/cli'
+
+# rubocop:disable Metrics/BlockLength
+module DangerUndercover
+  describe DangerUndercover::CLI do
+    let!(:mock_message) { 'Test Passed' }
+    let!(:directory) { File.join(Dir.getwd, 'coverage') }
+    let!(:file) { File.join(Dir.getwd, 'coverage/undercover.txt') }
+
+    before(:each) do
+      allow(described_class).to receive(:`).and_return(mock_message)
```

Comment:
What is `receive(:`)` for?

Reply:
`receive(:`)` is to mock the - 

`undercover #{args&.join(' ')}` CLI command output.

Reply:
Wow, I would not have guessed. So a comment would help future devs, yourself included.